src/Form.vue

```diff
@@ -96,10 +96,18 @@
         &__checkboxes,
         &__checkbox,
         &__radios,
+        &__switch,
         &__date {
             display: block;
             margin: 2em 0;
 
+        &__checkboxes,
+        &__switch,
+        &__radios {
+            label > span {
+                display: inline;
+            }
+
```

Comment:
This block (lines 104-110) should be removed. It's not needed, and also is not valid scss syntax.